CHANGELOG.md

```diff
@@ -7,6 +7,29 @@ Mapbox welcomes participation and contributions from everyone.
 #### Bug fixes and improvements
 - Fixed an issue with `NavigationView` that caused info panel to shrink in landscape mode with a full screen theme. [#6811](https://github.com/mapbox/mapbox-navigation-android/pull/6811)
 - Fixed standalone `MapboxManeuverView` appearance when the app also integrates Drop-In UI. [#6810](https://github.com/mapbox/mapbox-navigation-android/pull/6810)
+- :warning: Updated the `NavigationView` default navigation puck asset. [#6678](https://github.com/mapbox/mapbox-navigation-android/pull/6678)
+
+  Previous puck can be restored by injecting `LocationPuck2D` with the `bearingImage` set to `com.mapbox.navigation.ui.maps.R.drawable.mapbox_navigation_puck_icon` drawable:
+  ```kotlin
+  navigationView.customizeViewStyles {
+      locationPuckOptions = LocationPuckOptions.Builder(context)
+          .defaultPuck(
+              LocationPuck2D(
+                  bearingImage = ContextCompat.getDrawable(
+                      context,
+                      com.mapbox.navigation.ui.maps.R.drawable.mapbox_navigation_puck_icon,
+                  )
+              )
+          )
+          .idlePuck(regularPuck(context))
+          .build()
+  }
+  ```
+- Added guarantees that route progress with `RouteProgress#currentState == OFF_ROUTE` arrives earlier than `NavigationRerouteController#reroute` is called. [#6764](https://github.com/mapbox/mapbox-navigation-android/pull/6764)
+- Fixed a rare `java.lang.NullPointerException: Attempt to read from field 'SpeechAnnouncement PlayCallback.announcement' on a null object reference` crash in `PlayCallback.getAnnouncement`. [#6760](https://github.com/mapbox/mapbox-navigation-android/pull/6760)
+- Fixed standalone `MapboxManeuverView` appearance when the app also integrates Drop-In UI. [#6774](https://github.com/mapbox/mapbox-navigation-android/pull/6774)
+- Introduced `NavigationViewListener.onSpeedInfoClicked` that would be triggered when `MapboxSpeedInfoView` is clicked upon. [#6770](https://github.com/mapbox/mapbox-navigation-android/pull/6770)
+- Each newly instantiated MapboxRouteArrowView class will initialize the layers with the provided options on the first render call. Previously this would only be done if the layers hadn't already been initialized.  [#6466](https://github.com/mapbox/mapbox-navigation-android/pull/6466)
```

Comment:
Just like in https://github.com/mapbox/mapbox-navigation-android/pull/6815#discussion_r1066161453 you have to clean up the changelog. That always happens with cherry-picks, but that won't be an issue once we adopt the new changelog strategy by @SevaZhukov.

Reply:
This will lead to a conflict and a rebase will create a mess just like the last one.  Here we go...

Reply:
Fortunately no conflict this time.